bundle/tests/scorecard/kuttl/network-policy/03-assert.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: network-policy-rc
+spec:
+  ingress:
+    - from:
+        - namespaceSelector: {}
+      ports:
+        - port: 8080
```

Comment:
Kuttl wouldn't assert that 9090 and 9091 are no longer specified. We need an 'error' check on those ports to make sure they no longer exist.